src/d3a/models/area/markets.py

```diff
@@ -60,6 +60,10 @@ def _market_rotation(self, current_time, markets, past_markets, area_agent):
             if timeframe < current_time:
                 market = markets.pop(timeframe)
                 market.readonly = True
+                if ConstSettings.GeneralSettings.KEEP_PAST_MARKETS is False:
```

Comment:
You can use the 'not' operator instead of checking if it is False.

Reply:
@spyrostz Are there any specific advantage of using _not_ rather than _False_ ?

Reply:
There is a slight difference, `not` catches both `False` and `None`. So what I wanted to indicate (nitpicking, I know) is that if a user assigns something like `KEEP_PAST_MARKETS=None` out of ignorance of the parameter type, the program will treat it as False.